kafka-rest/src/main/java/io/confluent/kafkarest/resources/TopicsResource.java

```diff
@@ -136,6 +136,60 @@ public void produceAvro(
     produce(asyncResponse, topicName, EmbeddedFormat.AVRO, request);
```

Comment:
All three methods have basically the same implementation. Can you Extract-Method?